config/Version.public.xcconfig

```diff
@@ -1,4 +1,4 @@
 VERSION_SHORT=16.4
 
 // Public long version example: VERSION_LONG=9.9.0.0
-VERSION_LONG=16.4.0.2
+VERSION_LONG=16.4.0.5
```

Comment:
Does this mean we forgot to merge after betas .3, .4 and .5? pfew!

Reply:
Haha no – the release finalization failed the first time (burning `.3`) because of the duplicate milestone issue you'd mentioned (I didn't think this would cause it to fail, but guess I was wrong!)

So I fixed that and ran it again, and it failed because the frozen tag wasn't applied to the milestone anymore (burning `.4`). 

So I fixed _that_ and it ran almost all the way though (using `.5`), then finished it from there by hand by closing up the milestone.

Ideally I think this process would be idempotent, but this whole process only took 5 minutes, so it may not be worth the time to change it 😄

Reply:
Ah makes sense, thanks for the explanation!